pkg/agent/plugin/workloadattestor/windows/windows_windows_test.go

```diff
@@ -104,6 +110,43 @@ func TestAttest(t *testing.T) {
 			},
 			expectCode: codes.OK,
 		},
+		{
+			name: "successful getting path and hashing process binary",
```

Comment:
can you add  the next test cases:
- failed to get binary path
- failed to hash binary
- binary exceeds limit size